apps/studio/components/interfaces/Functions/EdgeFunctionSecrets/AddNewSecretModal.tsx

```diff
@@ -16,27 +21,44 @@ const AddNewSecretModal = ({ visible, onClose }: AddNewSecretModalProps) => {
   const submitRef = useRef<HTMLButtonElement>(null)
   const [showSecretValue, setShowSecretValue] = useState(false)
 
+  const FormSchema = z.object({
+    name: z
+      .string()
+      .min(1, 'Please provide a name for your secret')
+      .refine((value) => !value.match(/^(SUPABASE_).*/), {
```

Comment:
Are we already validating that these secrets are uppercase? Am I allowed to create secrets with `supabase_` (lowercase)?

Reply:
these are the checks that API does on their end, so gonna follow them here (so yup lowercase should be fine)
https://github.com/supabase/infrastructure/blob/develop/api/src/routes/v1/projects/ref/secrets/secrets.controller.ts#L106
https://github.com/supabase/infrastructure/blob/develop/api/src/routes/v1/projects/ref/secrets/secrets.dto.ts#L10